server/src/test/java/org/elasticsearch/VersionTests.java

```diff
@@ -345,7 +345,7 @@ public static void assertUnknownVersion(Version version) {
 
     public void testIsCompatible() {
         assertTrue(isCompatible(Version.CURRENT, Version.CURRENT.minimumCompatibilityVersion()));
-        assertFalse(isCompatible(Version.fromString("6.7.0"), Version.fromString("7.0.0")));
+        assertFalse(isCompatible(Version.V_7_0_0, Version.V_8_0_0));
```

Comment:
We dont' want to have these constants used in tests like this because it requires updating each time we bump master. However, it isn't hurting anything here, so I will clean this up in followups.